scripts/copy_files.js

```diff
@@ -0,0 +1,17 @@
+const fs = require("fs");
+const path = require("path");
+
+// Define the source and destination file paths
+const sourceFile = path.join(__dirname, '../src/external/aria');
+const destinationFile = path.join(__dirname, '../lib/external/aria');
```

Comment:
I'd suggest to rewrite this to be less platform specific:

`path.join(path.dirname(__dirname), "lib", "external", "aria")`